Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -42,3 +42,45 @@
     from FastSimulation.Configuration.DigiAliases_cff import loadDigiAliases
     loadDigiAliases(premix = False)
     from FastSimulation.Configuration.DigiAliases_cff import generalTracks,ecalPreshowerDigis,ecalDigis,hcalDigis,muonDTDigis,muonCSCDigis,muonRPCDigis
+
+#phase 2 common mods
+def _modifyDigitizerPhase2Common( theProcess ):
+    theProcess.load("CalibCalorimetry/HcalPlugins/Hcal_Conditions_forGlobalTag_cff")
+    theProcess.es_hardcode.toGet = cms.untracked.vstring(
```

Comment:
@Dr15Jones @lgray - HCAL will deal with the hardcode condition loading later. #13681 needs to get merged first before we make further changes to that code.